src/native.js

```diff
@@ -0,0 +1,135 @@
+import { getBidRequest, logError } from './utils';
+
+/** INSERT NATIVE ADAPTERS - DO NOT EDIT OR REMOVE */
+const nativeAdapters = [];
+/** END INSERT NATIVE ADAPTERS */
```

Comment:
Writing it to adaptermanager.js like the other loader does causes a circular dependency. Can move it though if there's another place that makes more sense